src/kaleidoscope/key_defs.h

```diff
@@ -202,6 +203,45 @@ typedef kaleidoscope::Key Key_;
 #define ___ Key_Transparent
 #define XXX Key_NoKey
 
+namespace kaleidoscope {
+constexpr bool isKeyboardKey(Key key) {
+  return ((key.getFlags() & (RESERVED | SYNTHETIC)) == 0);
+}
+constexpr bool isKeyboardModifier(Key key) {
```

Comment:
I think this makes most sense as "isModifierKey" - as part of that, I think we should affirm that "Modifier"  always means  one of the keys between HID_KEYBOARD_FIRST_MODIFIER and HID_KEYBOARD_LAST_MODIFIER, inclusive.

Reply:
Okay, but I desperately want a term that encompasses both Keyboard HID modifiers and layer shift keys, since they are logically equivalent from the user's point of view. There's an existing function (whose name should probably change regardless) called `isModifierKey()` in Qukeys that does exactly that, grouping those two together.